content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md

```diff
@@ -16,207 +16,255 @@ aliases:
-
-{{< caution >}}
-**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later and you will have to generate a new one.
-{{< /caution >}}
-
-{{< img src="RecordClientIDandSecret.png" alt="Rectangle" caption="<em>Record your Client ID and Secret</em>" class="border-0" >}}
-
-5. Now that you have created your Custom Integration, follow the steps below to [test generating a token](./#getting-a-token-with-your-app-registration) and [ensure you can access the RKVST API](./#testing-your-access).
-
 {{< note >}}
-**Note:** By default, newly created Integrations will always have a Non-Administrator permission to the API, you must add the Integration as an Administrator to elevate it's permissions.
-
-You can add a Custom Integration as an Administrator using the `Settings` screen, where the issuer will be `https://app.rkvst.io/appidpv1` and the subject will be your Integration's `CLIENT_ID`.
+**Note:** Creating App Registrations requires **Administrator** privileges.  
+If `Settings` does not appear in the navigation, see your RKVST Administrator for access.
 {{< /note >}}
 
-### Using the App Registrations API to create an Integration
+### Using the RKVST App to Create an Integration (First-Time Setup)
+
+1. As an Administrator, open the <a href="https://app.rkvst.io/" target="_blank">RKVST App</a>
+1. Navigate to `Settings` on the sidebar
+1. Navigate to the `Integrations` tab
+  {{< img src="IntegrationsTab.png" alt="Rectangle" caption="<em>Navigate to Settings, then Integration</em>" class="border-0" >}}
+1. Click the `Custom` box to create a Custom Integration
+1. Enter any `Display Name` you'd like
+  {{< img src="Confirm.png" alt="Rectangle" caption="<em>Completed Web Registration</em>" class="border-0" >}}
+  {{< note >}}
+  Optionally add any `Custom claims` at this step by clicking the `+ Add` button. Ensure the `Name` _does not start_ with `jit_` (RKVST reserved names) or use any other [well-known reserved claims](https://auth0.com/docs/security/tokens/json-web-tokens/json-web-token-claims#reserved-claims).
+  {{< /note >}}  
+1. Once complete, click `Confirm` to complete the custom integration
+1. You will then be presented with the `CLIENT_ID` and `SECRET` required by the archivist token endpoint
+    {{< caution >}}
+    **Caution:** Save the `CLIENT_ID` and `SECRET` to a password manager or secret management service as the `SECRET` can **not** be viewed again. A new `SECRET` can be regenerated by editing the Integration, invalidating the previous `SECRET`.
+    {{< /caution >}}  
+    {{< img src="RecordClientIDandSecret.png" alt="Rectangle" caption="<em>Record your Client ID and Secret</em>" class="border-0" >}}
+2. Copy the `CLIENT_ID` and `SECRET` to a password manager for future reference
+3. Skip to [Grant Permissions to Custom Integration](#grant-permissions-to-custom-integration) to configure an Access Policy
+
+### Using the App Registrations API to Create an Integration
```

Comment:
I agree with Jon that this section should be deleted. I've never used it and the article flows much better without the interruption.